app/controllers/health_cards_controller.rb

```diff
@@ -11,6 +11,19 @@ class HealthCardsController < ApplicationController
   def show
     respond_to do |format|
       format.healthcard { render json: @exporter.download }
+      format.html do
+        @jws_encoded_details = @exporter.jws
+        @health_card = HealthCards::COVIDHealthCard.from_jws @jws_encoded_details
+        @qr_code_payload = @exporter.chunks
+
+        if @qr_code_payload.length == 1
+          @qr_code_payload[0] = "shc:/#{@qr_code_payload[0]}"
```

Comment:
I believe we should remove the quotes and brackets when we display this. Have the display start with `shc:/` instead of `["shc:/`. When I tested this with the Microsoft Verifier portal, I had to delete both of those to get it to decode. This also aligns with [example f](https://smarthealth.cards/examples/example-00-f-qr-code-numeric-value-0.txt) from the spec.